mbuild/compound.py

```diff
@@ -75,6 +76,12 @@ def load(filename, relative_to_module=None, compound=None, coords_only=False,
     if compound is None:
         compound = Compound()
 
+    # Handle the case of a xyz file, which must use an internal reader
+    extension = os.path.splitext(filename)[-1]
+    if extension == '.xyz':
```

Comment:
We might want this to be `if extension == '.xyz' and not 'top' in kwargs:`. That way we can still use MDTraj's XYZ reader if a topology is also provided (and we should have unit tests covering both cases).